packages/ui/src/components/createInitiative.js

```diff
@@ -0,0 +1,134 @@
+/**
+ * @author Sandhya Salian
+ * @name Create Initiative
+ * @desc This component renders the create initiative page
+ */
+import React from 'react';
+import CreateInitiativeForm from './createInitiativeForm.js';
+
+class CreateInitiative extends React.Component {
+
+    /**
+     * Class constructor.
+     */
+    constructor(props) {
+        super(props);
+
+        // set the initial component state
+
+        this.state = {
+            errors: {},
+            user: {
+                initiativeName: '',
+                approverName: ''
+            },
+            totalParticipants: {}
+        };
+
+        this.processForm = this.processForm.bind(this);
+        this.changeUser = this.changeUser.bind(this);
+
+    }
+    handleValidation() {
+        let fields = this.state.user;
+        let errors = {};
+        let formIsValid = true;
+
+        if (!fields["initiativeName"]) {
+            formIsValid = false;
+            errors["initiativeName"] = "Enter Initiative Name";
```

Comment:
This messages should come from a constants, create a message.js file in constants and import all the constants from there